### PR TITLE
Convert tables to 2 column layout when less than 769px width

### PR DESCRIPTION
**Pull request by sg00dwin**
And update 2 broken paths to their correct locations within patternfly components js

### CODE REVIEW

**Comment by sg00dwin** [comment]
pods
services
image-streams
deployments
build-config
builds
deployment-config
deployment
routes

File: assets/app/views/services.html [hunk 1]
```diff
@@ -5,46 +5,46 @@
...
+          </tr>
+        </thead>
+        <tbody ng-if="(services | hashSize) == 0">
+          <tr><td colspan="6"><em>{{emptyMessage}}</em></td></tr>
+        </tbody>
+        <tbody ng-repeat="service in services | orderObjectsByDate : true">
+          <tr>
+            <td data-title="Name"><a href="project/{{projectName}}/browse/services/{{service.metadata.name}}">{{service.metadata.name}}</a></td>
+            <td data-title="Cluster IP">{{service.spec.clusterIP}}</td>
+            <td data-title="EXternal IP">
```
Comment (jwforres) [inline assets/app/views/services.html]: typo, lower case the X

File: assets/app/views/services.html [hunk 2]
```diff
@@ -5,46 +5,46 @@
...
-          </tbody>
-        </table>
-      </div>
+      <table class="table table-bordered table-hover table-mobile">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Cluster IP</th>
+            <th>External IP</th>
+            <th>Ports</th>
```
Comment (sg00dwin) [inline assets/app/views/services.html]: Fixed type

Comment (sg00dwin) [inline assets/app/views/services.html]: Fixed typo ready for merge
https://github.com/openshift/origin/pull/5087/files#diff-9d3529dae36b903d64ba7957d01e5d32

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
@sg00dwin merge conflict. probably with the final version of my changes related to history removal

**Comment by sg00dwin** [comment]
@jwforres rebased and updated

**Comment by jwforres** [comment]
[merge]

**Pull request merged**